README.md

```diff
@@ -1,2 +1,93 @@
+
+For information on the sandbox process please check out the [details on our sandbox application process below](#how-applications-are-reviewed).
+
+## How to apply
+
+Applying is as easy as 1-2-3!
+
+1. Open a new [Sandbox Application](https://github.com/cncf/sandbox/issues/new) using the linked issue form. 
+1. Complete all sections of the form.
+1. Submit the issue.
+
+## What's next?
+
+Your project's application is placed in the backlog for triaging. You can view the status of your application and others at anytime by checking out our [Project Board](https://github.com/orgs/cncf/projects/14).
+
+When the TOC meets to review applications, more details on [frequency here](#frequency), your application will be pulled up and discussed. Your application may receive one of several statuses:
+* `New` - It is a brand new application and is in the backlog for an upcoming review.
+* `Approved` - The application has been reviewed and been approved by the TOC. The issue will be closed.
+* `Declined` - The application has been reviewed and been declined by the TOC. The comments on the issue will reflect the TOC decision and the issue will be closed.
+* `Need-Info` - The application has been reviewed and requires more information in order for the TOC to further discuss the application.  For instructions on what to do, please jump down to [Need-Info](#Need-Info).
+* `TAG-Assigned` - The application has been reviewed and the TOC would like a TAG to review or consult with the project prior to further discussions.  For instructions on what to do, please jump down to [TAG-Assigned](#TAG-Assigned).
+* `Postponed` - The application has been reviewed and the TOC has determined the project (as it exists at time of review) is not ready for inclusion within the CNCF. For instructions on what to do, please jump down to [Postponed](#Postponed)
+* `Returning` - The application has been reviewed previously, was affixed a `Need-Info` or `TAG-Assigned` label, and that work has been completed and is ready for re-review.  For instructions on what to do, please jump down to [Returning](#Returning).
+
+### Label statuses defined
+
+#### Need Info
+
+If your project is assigned a `Need-Info` label, the TOC will comment on the issue with the specific additional information needed.  It will then be placed in the `Waiting` status of the [Project Board](https://github.com/orgs/cncf/projects/14).
+
+Please provide the additional information requested as a comment on the issue. Once you have done this, **remove the `Need-Info` label and affix the `Returning` label**, this lets the TOC know it is ready to be reviewed again. It will then be removed from the `Waiting` status and placed in the `Upcoming` status for discussion at the next meeting.
```

Comment:
Who has the permissions to change labels on application issues?  Can the submitter do this, or does a member of the TOC or staff need to?

Reply:
"Anyone with triage access to a repository can apply and dismiss labels." per https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels

Reply:
Will the issue creator be able to do that, though?   The above workflow seems to depend on them doing so.

We'll also need instructions for how to handle it if the person handling the "returning" isn't the one who filed the original sandbox issue.

Reply:
I spent some time working through the automation required, and ended up that a TOC member and/or CNCF staff will have to do this, the original submitter can provide comments.

Reply:
Returning: reopen an issue with comments?